packages/libs/react-ui/src/components/Button/Button.tsx

```diff
@@ -19,7 +19,9 @@ export interface IButtonProps
   icon?: keyof typeof SystemIcon;
   iconAlign?: 'left' | 'right';
   loading?: boolean;
-  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+  onClick?:
+    | React.MouseEventHandler<HTMLButtonElement>
+    | React.FormEventHandler<HTMLButtonElement>;
```

Comment:
@eileenmguo can you have a look at the ReactUI component changes?
or are these already discussed?